tests/unit/extend/find-many-test.js

```diff
@@ -0,0 +1,50 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import ApplicationAdapter from "../../helpers/properties/application-adapter";
+import { create } from 'ember-cli-page-object';
+import { findMany } from 'ember-cli-page-object/extend';
+import { findAll } from '@ember/test-helpers';
+
+module(`Extend | findMany`, function(hooks) {
+  setupApplicationTest(hooks);
+
+  let adapter = new ApplicationAdapter(hooks);
+
+  test('finds by selector and returns array of Elements', async function(assert) {
+    let page = create({});
+
+    await adapter.createTemplate(this, page, '<em class="lorem"></em><span class="ipsum"></span>');
```

Comment:
since at the end we should not care if about an actual execution context used, maybe we can use regular Ember testing APIs: `setupRenderingTest`, `await render`...?